charts/openmetadata/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.networkPolicy.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: {{ include "OpenMetadata.fullname" . }}-networkpolicy
+  namespace: {{ .Release.Namespace | quote }}
```

Comment:
This is not required. Let's remove this! Helm install / upgrade will take care of deploying this manifest in the relevant namespaces where openmetadata is being installed!

Reply:
Noted.